resources/public/api/conf/2.0/application.yaml

```diff
@@ -165,14 +165,18 @@ paths:
             example: 3CnsTh79I7vtOy6
         - in: query
           name: page
-          description: This is an optional query parameter, default is 1, that allows the selection of a specific page from a multi-page query.
+          description: >
+            This is an optional query parameter, default is 1, that allows the selection of a specific page from a multi-page query.
+            A check is made on the page parameter to ensure it is a positive number.

```

Comment:
Suggestion:
This is an optional query parameter (default value is `1`) that allows the selection of a specific page from a multi-page query. If used, you must enter a positive number.